package.json

```diff
@@ -9,18 +9,18 @@
   "scripts": {
     "postinstall": "npx patch-package && cd desktop && npm install",
     "clean": "npx react-native clean-project-auto",
-    "android": "scripts/set-pusher-suffix.sh &&  concurrently \"npx react-native run-android --port=8083\" npm:prettier-watch",
-    "ios": "scripts/set-pusher-suffix.sh && concurrently \"npx react-native run-ios --port=8082\" npm:prettier-watch",
-    "ipad": " concurrently \"npx react-native run-ios --port=8082 --simulator=\"iPad Pro (12.9-inch) (4th generation)\"\" npm:prettier-watch",
-    "ipad-sm": " concurrently \"npx react-native run-ios --port=8082 --simulator=\"iPad Pro (9.7-inch)\"\" npm:prettier-watch",
+    "android": "scripts/set-pusher-suffix.sh &&  concurrently \"npx react-native run-android --port=8083\"",
+    "ios": "scripts/set-pusher-suffix.sh && concurrently \"npx react-native run-ios --port=8082\"",
+    "ipad": " concurrently \"npx react-native run-ios --port=8082 --simulator=\"iPad Pro (12.9-inch) (4th generation)\"\"",
+    "ipad-sm": " concurrently \"npx react-native run-ios --port=8082 --simulator=\"iPad Pro (9.7-inch)\"\"",
```

Comment:
Why do we have these changes?

Reply:
I did not intend to commit them, this was me disabling prettier because it was fucking up my changes otherwise. 
I will revert